src/store/models/taro.ts

```diff
@@ -123,10 +129,19 @@ const taroModel: TaroModel = {
         blocks: BLOCKS_TIL_CONFIRMED,
         node: btcNode,
       });
-
       return res;
     },
   ),
+  getNewAddress: thunk(
+    async (actions, { node, genesisBootstrapInfo, amount }, { injections }) => {
+      const api = injections.taroFactory.getService(node);
+      const address = await api.newAddress(node, {
+        genesisBootstrapInfo: Buffer.from(genesisBootstrapInfo as string, 'hex'),
```

Comment:
The ` as string` can be removed since the type is already set to `string` on `NewAddressPayload`

Reply:
done